web/src/components/documentations/DocumentationCreateForm.vue

```diff
@@ -78,10 +82,21 @@ const documentation = ref<Partial<Documentation>>({})
 const formRef = ref<InstanceType<typeof VForm> | null>(null)
 
 const isCreating = ref(false)
+const isValid = ref(false)
```

Comment:
@burkkyy I think you can do this with <v-form v-model="isValid">? See https://vuetifyjs.com/en/api/v-form/#props-model-value and first example source code at https://vuetifyjs.com/en/components/forms/#usage